include/nbl/video/IPhysicalDevice.h

```diff
@@ -136,13 +142,33 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
         assert(false); // Invalid code path
         return false;
     }
+    
+    virtual bool isRayTracingPipelineSupported() const
+    {
+        assert(false); // Invalid code path
+        return false;
+    }
+
+    virtual bool isAccelerationStructuresSupported() const
+    {
+        assert(false); // Invalid code path
+        return false;
+    }
 
+    virtual bool isRayQuerySupported() const
+    {
+        assert(false); // Invalid code path
+        return false;
+    }
```

Comment:
I would have thought these would correllate to the booleans?

P.S. Dont assert here, the user is only asking for availability not actually using the extensions

Reply:
Understood, I just followed what `isSwapchainSupported` did
I guess I should remove the assert from that function as well?

Reply:
yes please